### ISSUE TITLE
Error: unrecognized identifier 'DEFAULT_UNITY_VERTEX_OUTPUT_STEREO_EYE_INDEX'

### ISSUE THREAD

**Issue by a-iafrate** [microsoft/MixedRealityToolkit-Unity#8854]
## Describe the bug

On an Android project that use MRTK  compile with il2cpp falils with error
unrecognized identifier 'DEFAULT_UNITY_VERTEX_OUTPUT_STEREO_EYE_INDEX'

## To reproduce

Create a unity project for android with mrtk
Set il2cpp as scripting backend
Add arm64 target
Build

## Expected behavior

Have the apk file

## Screenshots

![image](https://user-images.githubusercontent.com/8555750/97373481-64409280-18b6-11eb-8cfa-47128cdc9f1f.png)

## Your setup (please complete the following information)

- Unity Version 2019.4.11f1
- MRTK Version 2.5.1

## Target platform (please complete the following information)

- Android

Complete error:

```
Shader error in 'VR/SpatialMapping/Wireframe': unrecognized identifier 'DEFAULT_UNITY_VERTEX_OUTPUT_STEREO_EYE_INDEX' at line 38 (on gles3)

Compiling Vertex program with STEREO_MULTIVIEW_ON
Platform defines: UNITY_NO_DXT5nm UNITY_NO_RGBM UNITY_ENABLE_REFLECTION_BUFFERS UNITY_FRAMEBUFFER_FETCH_AVAILABLE UNITY_NO_CUBEMAP_ARRAY UNITY_NO_SCREENSPACE_SHADOWS UNITY_PBS_USE_BRDF2 SHADER_API_MOBILE UNITY_HARDWARE_TIER3 UNITY_COLORSPACE_GAMMA UNITY_LIGHTMAP_DLDR_ENCODING
```

Thanks for help

**Comment by keveleigh** [comment]
Unfortunately, I don't believe `VR/SpatialMapping/Wireframe` is an MRTK shader, but one Unity ships. You might want to file a bug report to Unity and/or post on their forums.

**Comment by AmineDev20** [comment]
I have the same issue i cant fix it

**Comment by keveleigh** [comment]
> I have the same issue i cant fix it

Unfortunately, this isn't an MRTK shader. You'll want to report the issue on the Unity forums and ask them to update.
Which version of Unity are you using?

**Comment by DaamHaam** [comment]
Hi, I have the same problem
Shader error in 'VR/SpatialMapping/Wireframe': unrecognized identifier 'DEFAULT_UNITY_VERTEX_OUTPUT_STEREO_EYE_INDEX' at line 38 (on gles3)

Using Unity 2021.3.4f1LTS 
Android build

**Comment by amigax** [comment]
same! how do we delete the shader?

**Comment by RaghavVenkat** [comment]
I resolved this by updating the Unity Editor to the 2023 LTS version. In parallel, I was also updating my Visual Studio Editor during which this showed up 

![image](https://github.com/microsoft/MixedRealityToolkit-Unity/assets/17124794/7285fc69-f1d5-43a3-860f-c55b002ba77b)

Just thought of sharing this image for everyones reference to investigate further on this issue.

**Comment by IssueSyncBot** [comment]
We appreciate your feedback and thank you for reporting this issue.

Microsoft Mixed Reality Toolkit version 2 (MRTK2) is currently in limited support. This means that Microsoft is only fixing high priority security issues. Unfortunately, this issue does not meet the necessary priority and will be closed. If you strongly feel that this issue deserves more attention, please open a new issue and explain why it is important.

Microsoft recommends that all new HoloLens 2 Unity applications use MRTK3 instead of MRTK2.

Please note that MRTK3 was released in August 2023. It features an all-new architecture for developing rich mixed reality experiences and has a minimum requirement of Unity 2021.3 LTS. For more information about MRTK3, please visit [https://www.mixedrealitytoolkit.org](https://www.mixedrealitytoolkit.org).

Thank you for your continued support of the Mixed Reality Toolkit!
